src/Stryker.Core/Stryker.Core/Reporters/Json/SourceFiles/JsonMutant.cs

```diff
@@ -38,6 +38,8 @@ public JsonMutant(IReadOnlyMutant mutant)
             StatusReason = mutant.ResultStatusReason;
 
             Static = mutant.IsStaticValue;
+
+            CoveredBy = mutant.CoveringTests.GetGuids()?.Select(g => g.ToString());
```

Comment:
Can null happen there? I'd expect an empty collection if nothing is found.

Reply:
I think there is a difference between there's no test information and there's no test covering this mutant.

Reply:
Well it's just a technical question - I was a bit lazy to look into the implementation of the GetGuids, if it even can return nulls :)